src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java

```diff
@@ -208,4 +209,42 @@ public void reinstallApicurito() {
         ApicuritoTemplate.cleanNamespace();
         ApicuritoTemplate.reinstallApicurito();
     }
+
+    @Then("check that metering labels have correct values")
+    public void checkThatMeteringLabelsHaveCorrectValues() {
```

Comment:
Please split the checks for each component. Similar to the method for checking running pods.